src/effects/native/filtereffect.cpp

```diff
@@ -1,8 +1,10 @@
 #include "effects/native/filtereffect.h"
 #include "util/math.h"
 
-static const double kMinCorner = 0.0003; // 13 Hz @ 44100
-static const double kMaxCorner = 0.5; // 22050 Hz @ 44100
+namespace {
+static const double kMinCorner = 13; // Hz
+static const double kMaxCorner = 22050; // Hz
```

Comment:
static keyword = old style anonymous namespace = new style. So static is redundant now.